examples/embed/file_html.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import print_function
```

Comment:
This isn't needed anymore (a Py2 thing).

Reply:
Thanks -